src/js/object-mona-parser/object-mona-parser.d.ts

```diff
@@ -56,8 +56,10 @@ export interface AlgebraicTypeParseResult {
 
 export interface ParsedAttributeType {
   isNSObject:boolean;
+  name:string;
   reference:string;
   underlyingType:string;
+  conformingProtocol:string;
```

Comment:
There can be multiple protocols to conform to, we should probably loop back and make this an array at some point